packages/web3torrent/src/utils/torrent-status-checker.test.ts

```diff
@@ -1,16 +1,18 @@
 import {web3torrent} from '../clients/web3torrent-client';
 import {Status, Torrent} from '../types';
 import {createMockTorrent} from './test-utils';
-import checkTorrentStatus, {getFormattedETA, getStatus} from './torrent-status-checker';
+import torrentStatusCheck, {getFormattedETA, getStatus} from './torrent-status-checker';
```

Comment:
Maybe get rid of the default export `torrentStatusChecker` to keep import style consistent?

Reply:
sure